go-controller/pkg/informer/informer_test.go

```diff
@@ -261,8 +261,8 @@ var _ = Describe("Informer Event Handler Tests", func() {
 
 		_, err := k.CoreV1().Pods(namespace).Update(context.TODO(), pod, metav1.UpdateOptions{})
 		Expect(err).NotTo(HaveOccurred())
-		// no deletes
-		Consistently(func() int32 { return atomic.LoadInt32(&deletes) }).Should(Equal(int32(0)), "deletes")
+
+		Eventually(func() int32 { return atomic.LoadInt32(&deletes) }).Should(Equal(int32(0)), "deletes")
```

Comment:
@trozet the // no deletes comment would imply we actually *do* want consistently, no?

Reply:
Yeah for this case Consistently would make sense. I just blindly changed them all. I'll fix this and check the other cases.

Reply:
basically I think that all that checks for a _zero_ value should be `Consistently` and a _non-zero_ `Eventually`, we are never decrementing though

Reply:
now I'm seeing the race fail in a different case than the original. I'm just going to fix them 1 by 1 and keep retesting in this PR.